### DISCUSSION TITLE
Consider setting up a Liberpay account

### DISCUSSION THREAD

**Discussion by thibaultmol** [fail2ban/fail2ban#3795] [Ideas]
It's a donation platform that allows for very flexible payment and you only pay transaction fees. (in EU you it even supports SEPA transactions).
it's used by a decent amount of people/projects and is open source;
https://liberapay.com/

(I personally use it to donate to several people/projects)

**Comment by sebres** [comment]
Here is Liberapay of me: <a href="https://liberapay.com/sebres/donate"><img alt="Donate using Liberapay" src="https://liberapay.com/assets/widgets/donate.svg" valign="middle"></a>

We had already several discussions for general fail2ban sponsoring, e. g. #2818 and #1679, but they have never been realized, so at the moment it is what it is... Just for the record, to link them together.

But thanks for the remainder, I'll extend the FUNDING now, at least for my humble self.

**Comment by IgorLytkin** [comment]
For Russia - can you use https://boosty.to/ ?
